js/apps/admin-ui/src/realm-settings/EmailTab.tsx

```diff
@@ -33,18 +33,20 @@ import "./realm-settings-section.css";
 
 type RealmSettingsEmailTabProps = {
   realm: RealmRepresentation;
+  save: (realm: RealmRepresentation) => void;
 };
 
 export const RealmSettingsEmailTab = ({
   realm: initialRealm,
+  save,
 }: RealmSettingsEmailTabProps) => {
   const { t } = useTranslation("realm-settings");
   const { adminClient } = useAdminClient();
   const { realm: realmName } = useRealm();
   const { addAlert, addError } = useAlerts();
   const currentUser = useCurrentUser();
+  const [realm] = useState(initialRealm);
```

Comment:
Since this value is always passed from higher up there is no need to keep it in a state any longer.
```suggestion
```